packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -64,6 +67,60 @@ enum NavigationDecision {
   navigate,
 }
 
+/// Android [WebViewPlatform] that uses [AndroidViewSurface] to build the [WebView] widget.
+///
+/// To use this, set [WebView.platform] to an instance of this class.
+class SurfaceAndroidWebView extends AndroidWebView {
```

Comment:
Is the AndroidManifest change required? if so we should probably mention it here

Reply:
This change is no longer required, so I removed it from the changelog and example